presto/eval.py

```diff
@@ -171,7 +176,7 @@ def _inference_for_dl(
     def spatial_inference(
         self,
         finetuned_model: Union[PrestoFineTuningModel, BaseEstimator],
```

Comment:
Can this not be a `CatBoostClassifier`? (but see previous comment, maybe we need to map it to a `BaseEstimator` as well, or use our own implementation of a base classifier class.)

Reply:
Addressed with the `SklearnStyleClassifier`